submissions/YanaGushlevskaya/js-DOM/src/components/Hero/hero.js

```diff
@@ -1,2 +1,83 @@
+  ),
+  new cuisine(
+    'Грузинская кухня',
+    'img/georgian.png',
+    'georgian cuisine',
+    'Чанахи, харчо, лобио, чахохбили, соус ткемали, хачапури, цыплята табака, мацони, имеретинский сыр. Одно лишь это (конечно же, неполное) перечисление блюд и продуктов намекает на то, что речь пойдёт о грузинской кухне. Знаменитые блюда грузинского народа, почитаемые не только местными жителями, но завоевавшие любовь едоков всего мира, нельзя спутать, пожалуй, ни с какими другими. Употребление в пищу мяса разных сортов (говядины, баранины, свинины, мяса кур и индеек), чуть меньшее употребление рыбы характерны как для западной, так и восточной грузинской кухни. Широкое применение различных соусов делает блюда, приготовленные, казалось бы, из одинаковых продуктов, совершенно разными. И это – ещё одна особенность грузинской кухни, наравне с непременным употреблением свежих овощей, пряных трав и специй.'
+  ),
+  new cuisine(
+    'Итальянская кухня',
+    'img/italian.png',
+    'georgian cuisine',
+    'Разные области Италии привнесли и разные блюда в то их многообразие, что мы привыкли называть «итальянской кухней». Путешествуя по этой чудесной стране, можно отведать знаменитую пасту и равиоли, ризотто и удивительный итальянский десерт – тирамису. Блюда из свежей рыбы, овощные супы, большое количество маслин и, конечно же, разнообразные сорта сыра. Без добавления сыра, пожалуй, не обходится ни одно значимое блюдо итальянского обеда. Различные морепродукты, фрукты, приправы и душистые травы, рис, мясо и оливковое масло. Всего не перечесть!'
+  )
+];
 
+const cuisineTemplate = ({ title, src, alt, description }) => {
+  return `<h2 class="description__title">${title}</h2>
+      <figure>
+        <img src="${src}" alt="${alt}" class="decription__img" />
+        <figcaption>
+          <p class="description__img-description">
+          ${description}
+          </p>
+        </figcaption>
+      </figure>`;
+};
+
+const nav = document.querySelector('.nav');
+const menu = document.querySelector('.nav__list');
+const description = document.createElement('div');
+let windowWidth = window.innerWidth || document.documentElement.clientWidth || document.body.clientWidth;
+
+menu.addEventListener('click', createElement);
+
+function createElement({ target }) {
+  let button = target.textContent;
+  dataBase.forEach(cuisineItem => {
+    if (cuisineItem.title === button && windowWidth > 768) {
```

Comment:
For me it will be  apropos  to use const for `768` like `tabletResolution`.

Reply:
It is something strange, I really made a const for 768)) Maybe I deleted a wrong code. Thanks, I'll change it. I should install some tool for js validation.

Reply:
About page background,to avoid that behavior.

Reply:
@ArthurGorbenko Added the new background for mobile version - take a look)